lib/path.py

```diff
@@ -0,0 +1,812 @@
+import errno
```

Comment:
Maybe this module could be called `filesystem`?

Reply:
Good idea! This would also help resolve the naming conflict mentioned earlier.